packages/payload/src/auth/operations/logout.ts

```diff
@@ -36,5 +38,36 @@ export const logoutOperation = async (incomingArgs: Arguments): Promise<boolean>
     }
   }
 
+  if (collectionConfig.auth.disableLocalStrategy !== true && collectionConfig.auth.useSessions) {
+    const userWithSessions = await req.payload.db.findOne<{
+      id: number | string
+      sessions: { id: string }[]
+    }>({
+      collection: collectionConfig.slug,
+      req,
+      where: {
+        id: {
+          equals: user.id,
+        },
+      },
+    })
+
+    if (allSessions) {
```

Comment:
I don't follow what we're doing with the "allSessions" flag.

Reply:
logging out of all sessions